packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -607,8 +607,8 @@ function processOp(
 					type: string;
 				};
 				const address = envelope.address;
-				type = `${type}/${innerContent.type}`;
-				switch (innerContent.type) {
+				type = `${type}/${innerContent?.type}`;
```

Comment:
Looking this over, all this code only applies to `FluidDataStoreOp`.  I'd pull the other 3 to their own case chunk and have it do nothing or whatever makes sense.

You'll be able to remove several `as`'s (e.g. `as IEnvelope` on line 600) this way too, because the types express the type of `contents`

Reply:
And revert the addition of `?`